examples/ceed/ex2.h

```diff
@@ -103,20 +103,21 @@ CEED_QFUNCTION(f_apply_diff)(void *ctx, const CeedInt Q,
     CeedPragmaSIMD
     for (CeedInt i=0; i<Q; i++) {
       // Read spatial derivatives of u
-      const CeedScalar du[2]        =  {ug[i+Q*0],
-                                        ug[i+Q*1]
-                                       };
+      const CeedScalar du[2] = {ug[i+Q*0],
+                                ug[i+Q*1]
+                               };
 
       // Read qdata (dXdxdXdxT symmetric matrix)
       // Stored in Voigt convention
       // 0 2
       // 2 1
-      // *INDENT-OFF*
-      const CeedScalar dXdxdXdxT[2][2] = {{qdata[i+0*Q],
-                                           qdata[i+2*Q]},
-                                          {qdata[i+2*Q],
-                                           qdata[i+1*Q]}};
-      // *INDENT-ON*
+      const CeedInt idx[] = {0, 2, 2, 1};
+      CeedScalar dXdxdXdxT[2][2];
+      for (CeedInt m=0; m<2; m++)
+        for (CeedInt n=0; n<2; n++)
+          dXdxdXdxT[m][n] = qdata[i+idx[n+2*m]*Q];
```

Comment:
Are we sure this is the preferred way we want this? It seems to me that it is harder to read and therefore to spot bugs, than fixing indentation style by hand, if/when needed

Reply:
I'm happy with dropping 2f40c9d502a896ccbf8765d62c5bdd42b5843f70 and cfd0a8060ed6616b70ae77e2243fb3d74a7dd253 if everyone prefers, but without doing something `make style` is useless - fixing errors in 20+ files every time is not practical.

I think this comes back to the same discussion in https://github.com/CEED/libCEED/pull/323